src/orion/core/evc/experiment.py

```diff
@@ -117,6 +118,14 @@ def adapter(self):
         """Get the adapter of the experiment with respect to its parent"""
         return self.item.refers["adapter"]
 
+    @property
+    def tree_name(self):
+        """Return a formatted name of the Node for a tree pretty-print."""
+        if self.item is not None:
+            return self.name + " - v.{}".format(self.item.version)
```

Comment:
`"-v{}"` for compactness? It renders a bit loose with `pptree` with spaces. There isn't x.y versions, so the dot seems superfluous.